modules/rhel-images-aws.adoc

```diff
@@ -0,0 +1,47 @@
+// Module included in the following assemblies:
+//
+// * machine_management/adding-rhel-compute.adoc
+
+[id="rhel-images-aws_{context}"]
+= Listing latest available RHEL images on AWS
+
+AMI IDs correspond to native boot images for AWS. Because an AMI must exist before the EC2 instance is provisioned, you will need to know the AMI ID before configuration. The link:https://aws.amazon.com/cli/[AWS Command Line Interface (CLI)] is used to list the available {op-system-base-full} image IDs.
+
+.Prerequisites
+
+* You have installed the AWS CLI.
+
+.Procedure
+
+* Use this command to generate {op-system-base} 7.9 Amazon Machine Images (AMI):
+
+[source, terminal]
+----
+$ aws ec2 describe-images --owners 309956199498 \ <1>
+--query 'sort_by(Images, &CreationDate)[*].[CreationDate,Name,ImageId]' \ <2>
+--filters "Name=name,Values=RHEL-7.9*" \ <3>
+--region us-east-1 \ <4>
+--output table <5>
+----
+
+.Example output
+[source,terminal]
+----
+----------------------------------------------------------------------------------------------------------
+|                                             DescribeImages                                             |
++---------------------------+----------------------------------------------------+-----------------------+
+|  2020-05-13T09:50:36.000Z |  RHEL-7.9_HVM_BETA-20200422-x86_64-0-Hourly2-GP2  |  ami-038714142142a6a64 |
+|  2020-09-18T07:51:03.000Z |  RHEL-7.9_HVM_GA-20200917-x86_64-0-Hourly2-GP2    |  ami-005b7876121b7244d |
+|  2021-02-09T09:46:19.000Z |  RHEL-7.9_HVM-20210208-x86_64-0-Hourly2-GP2       |  ami-030e754805234517e |
++---------------------------+----------------------------------------------------+-----------------------+
+----
+<1> The `--owners` command option shows Red Hat images based on the account ID `309956199498`.
+<2> The `--query` command option sets how the images are sorted with the parameters `'sort_by(Images, &CreationDate)[*].[CreationDate,Name,ImageId]'`. In this case, the images are sorted by the creation date, and the table is structured to show the creation date, name of the image, and the AMI IDs.
+<3> The `--filter` command option sets what version of {op-system-base} is shown. In this example, since the filter is set by `"Name=name,Values=RHEL-7.9*"`, then {op-system-base} 7.9 AMIs are shown.
```

Comment:
```suggestion
<3> The `--filter` command option sets which version of {op-system-base} is shown. In this example, since the filter is set by `"Name=name,Values=RHEL-7.9*"`, then {op-system-base} 7.9 AMIs are shown.
```